packages/modal/src/Modal.spec.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { render, cleanup } from '@testing-library/react';
+import 'jest-dom/extend-expect';
+import Modal from './Modal';
+
+afterAll(cleanup);
+
+describe('packages/Modal', () => {
```

Comment:
Can we add a test for escape key? Both after a user types in one character and then the esc key?

Reply:
👍

Reply:
Removed the once only, so not sure we need both but happy to include